ironfish-cli/src/commands/service/sync-multi-asset.ts

```diff
@@ -198,14 +199,42 @@ export default class SyncMaspTransactions extends IronfishCommand {
   }
 }
 
-function serializeMasp(data: GetTransactionStreamResponse): ApiMaspUpload {
+function serializeMultiAssets(data: GetTransactionStreamResponse): ApiMultiAssetUpload {
+  const txs = []
+  // should not send transactions if block is disconnected
+  if (data.type !== 'connected') {
+    for (const tx of data.transactions) {
+      const multiAssets = []
+      for (const mint of tx.mints) {
+        multiAssets.push({
+          type: 'MULTI_ASSET_MINT' as MultiAssetTypes,
+          assetName: mint.assetName,
+        })
+      }
+      for (const burn of tx.burns) {
+        multiAssets.push({
+          type: 'MULTI_ASSET_BURN' as MultiAssetTypes,
+          assetName: burn.assetName,
+        })
+      }
+      for (const note of tx.notes) {
+        // standard notes should not be included
+        if (note.assetId !== Asset.nativeId().toString('hex')) {
```

Comment:
This piece isn't exactly right because you can also do a burn of the default asset. I or should be also added there too.

Reply:
Burns are not handled via the notes, they are handled via `tx.burns` which don't filter based on native asset. This is following the assumption that burns of the native asset will show up in `burns` rather than `notes`